packages/api/src/api/project/post.ts

```diff
@@ -0,0 +1,36 @@
+import { Schema } from '@hangar/shared';
+import { Request, Response } from 'express';
+import { logger } from '../../utils/logger';
+import { validatePayload } from '../../utils/validatePayload';
+import { Project, User } from '@hangar/database';
+
+export const post = async (req: Request, res: Response) => {
+  const { entityManager: em, user } = req;
+
+  const { errorHandled, data } = validatePayload({ req, res, schema: Schema.project.post });
+  if (errorHandled) return;
+  
+  const query = {id:req.user.id};
+  
+  // check for existing project
+  try {
+    const user = await em.findOne(User, query);
+    if (user?.project!==undefined) {
+      res.status(409);
+      return;
+    }
+  }
+  catch {
+    res.status(500);
+    return;
+  }
+  try {
+    const project = new Project(data);
+    em.persistAndFlush(project);
+    res.send(project);
+  }
+  catch(err) {
+    logger.error(err);
+    res.send(500);
```

Comment:
After creating the transaction above, you should check the error's status code and figure out if the error is an error due to the user being locked; if it is, throw a 409. [Example of how to check the status code](https://github.com/AmericanAirlines/Hangar/commit/07bfd5126b778012ed01361bbc7efa3f7e05376c#diff-c1aa0872df8ca3062b76e2375d28fa0961683b7790793fe0407ab6b113c5de8bL19-L24).